projects/ngqp/core/src/lib/types.ts

```diff
@@ -18,7 +18,7 @@ export type ParamDeserializer<T> = (value: string | null) => T | undefined | nul
  *
  * See {@link QueryParamOpts#combineWith}.
  */
-export type ParamCombinator<T> = (previousValue: T | undefined | null, newValue: T | undefined | null) => Params | null;
+export type ParamCombinator<T> = (newValue: T | undefined | null) => Params | null;
```

Comment:
Is it still a combinator? After this change it has just one value.

Reply:
Yeah, "combinator" isn't meant to refer combining old and current value, but rather combining the parameter change with changes to arbitrary other parameters.